ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -472,7 +480,6 @@ public void clear_aggregation()
         public bool neucode_heavy_lysine = false;
         public int max_ptms = 4;
         public int decoy_databases = 1;
-        public string decoy_database_name_prefix = "DecoyDatabase_";
```

Comment:
This was to make sure all of the prefixes were consistent, avoiding typo-related bugs.